server/server.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express');
+const bodyParser = require("body-parser");
 const path = require('path');
 const models = require('./models/index.js');
 const cors = require('cors');
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
This line could be replaced with `app.use(express.json())` for the same functionality and one less dependency